src/compile/data/optimize.ts

```diff
@@ -161,6 +161,7 @@ export function optimizeDataflow(dataComponent: DataComponent) {
 
   roots.forEach(moveFacetDown);
   roots.forEach(mergeParse);
+  roots.forEach(optimizers.mergeIdenticalTransforms);
```

Comment:
Hmm, why is the call pattern different from others (mergeParse isn't in optimizers?)

cc: @domoritz

Reply:
mergeParse should be in optimizers. It's not super clear because in a way it's necessary for correctness as well.